features/git-extract/cherry_pick_conflict/with_remote_origin/with_open_changes.feature

```diff
@@ -85,6 +80,13 @@ Feature: git extract: resolving conflicts between main branch and extracted comm
       |          |                  | refactor commit | conflicting_file |
       | refactor | local and remote | main commit     | conflicting_file |
       |          |                  | refactor commit | conflicting_file |
+    And now I have the following committed files
+      | BRANCH   | NAME             | CONTENT          |
+      | main     | conflicting_file | main content     |
+      | feature  | conflicting_file | refactor content |
+      | feature  | feature_file     |                  |
+      | refactor | conflicting_file | resolved content |
```

Comment:
When we have this, I don't think its necessary to have the files in the commit tables